q2_fmt/tests/test_engraftment.py

```diff
@@ -14,7 +14,7 @@
 from qiime2.plugin.testing import TestPluginBase
 from qiime2 import Metadata
 
-from q2_fmt._engraftment import group_timepoints
+from q2_fmt._engraftment import group_timepoints, _get_to_baseline_ref
```

Comment:
A thought I just had re: your refactoring of a couple of helper methods - you could group your helpers in a _utils.py file so that your actual private methods are separated from your public API. There's definitely some tradeoff there because you add another layer of indirection by the utils being in a different spot from the rest of your functionality - but it's also a pretty massive file right now, so it honestly might not add too much indirection. This is just a thought - not necessary to implement, but could be a helpful way to reorganize some code!

Reply:
Making an issue now! Thanks for the suggestion